### ISSUE TITLE
URL undefined undefined in release status

### ISSUE THREAD

**Issue by migmartri** [kubeapps/kubeapps#492]
![selection_647](https://user-images.githubusercontent.com/24523/44174193-253c1300-a097-11e8-951c-be485a2515af.png)

**Comment by absoludity** [comment]
Similar, but not the same. In #286, `status.loadBalancer.ingress[0].ip` is present, but set as Pending, whereas in this case, `status.loadBalancer.ingress[0].ip` is undefined in the svc resource, with `status.loadBalancer.ingress[0].hostname` being defined instead:
```
$ k get svc todo-nodejs-todo-with-ea-mean-template -o json | jq '.status.loadBalancer.ingress[0].ip'
null
~$ k get svc todo-nodejs-todo-with-ea-mean-template -o json | jq '.status.loadBalancer.ingress[0].hostname'
"internal-a5170830bb70c11e8bb33128041ffb7b-660769392.us-east-1.elb.amazonaws.com"
```

**Comment by andresmgot** [comment]
Thanks @absoludity. Raising the priority of this.

**Comment by andresmgot** [comment]
Closing this issue, it was fixed with #674
